packages/lua-language-server/build.sh

```diff
@@ -31,7 +31,8 @@ termux_step_host_build() {
 }
 
 termux_step_make() {
-	CFLAGS+=" -DBEE_ENABLE_FILESYSTEM" # without this, it tries to link against its own filesystem lib and fails.
+	CFLAGS+=" -DBEE_ENABLE_FILESYSTEM"     # without this, it tries to link against its own filesystem lib and fails.
+	CFLAGS+=" -Wno-unknown-warning-option" # for -Wno-maybe-uninitialized argument.
```

Comment:
It should be possible to remove this flag when a new NDK release is adopted which comes with a newer LLVM release. Adding a note for the same would be worth it

Reply:
Does clang even supports `-Wmaybe-uninitialized` flag? I couldn't find it even in [clang-15.0.0-git](https://clang.llvm.org/docs/DiagnosticsReference.html)

Reply:
Yeah, you're correct `-Wmaybe-uninitialized` isn't supported by clang. But `-Wuninitialized` is. Thanks for pointing out